packages/context-editor/src/utils/pubValues.ts

```diff
@@ -3,13 +3,13 @@ import { EditorState } from "prosemirror-state";
 
 export const getPubValues = (editorState: EditorState, pubId: string) => {
 	const editedPubs = {};
-	editedPubs[pubId] = {
-		id: pubId,
-		pubTypeId: "67704c04-4f04-46e9-b93e-e3988a992a9b",
-		values: {
-			"rd:content": editorState.doc.toJSON(),
-		},
-	};
+	// editedPubs[pubId] = {
+	// 	id: pubId,
+	// 	pubTypeId: "67704c04-4f04-46e9-b93e-e3988a992a9b",
+	// 	values: {
+	// 		"rd:content": editorState.doc.toJSON(),
+	// 	},
+	// };
```

Comment:
I'm not quite sure what we want to do here in practice—we will have a `RichText` pubfield saving the editor state—do we treat that as the "main content"? I commented this out for now since I think this could be done outside the editor